politeiad/backendv2/tstorebe/plugins/comments/cmds.go

```diff
@@ -701,6 +732,35 @@ func (p *commentsPlugin) cmdEdit(token []byte, payload string) (string, error) {
 	return string(reply), nil
 }
 
+// commentFirstVersion returns the first version of the specified comment. If
+// a comment is not found for the provided comment ID, a nil is returned.
+func (p *commentsPlugin) commentFirstVersion(token []byte, ridx recordIndex, commentID uint32) (*comments.Comment, error) {
+	cidx, ok := ridx.Comments[commentID]
+	if !ok {
+		// Comment does not exist
+		return nil, nil
+	}
+
+	// First version comment add digest
+	digest := cidx.Adds[1]
+
+	// Comment add record
+	adds, err := p.commentAdds(token, [][]byte{digest})
+	if err != nil {
+		return nil, errors.Errorf("commentAdds: %v", err)
+	}
+	if len(adds) != 1 {
+		return nil, errors.Errorf("wrong comment adds count; got %v, want %v",
+			len(adds), 1)
+	}
+
+	// Convert comment add
+	c := convertCommentFromCommentAdd(adds[0])
+	c.Downvotes, c.Upvotes = voteScore(cidx)
```

Comment:
i don't think we need to up/downvotes here, i can ditch them and mention that in
the function documentation.